frontend/packages/console-shared/src/components/pod/PodRing.tsx

```diff
@@ -67,8 +68,21 @@ const PodRing: React.FC<PodRingProps> = ({
     setClickCount(clickCount + operation);
     handleScaling(clickCount + operation);
   };
+
+  const {
+    apiVersion,
+    kind,
+    metadata: { name, namespace },
+  } = obj;
+  const [hpa] = useRelatedHPA(apiVersion, kind, name, namespace);
+  const hpaControlledScaling = !!hpa;
+
+  const isScalingAllowed = isAccessScalingAllowed && !hpaControlledScaling;
+
   const resourceObj = rc || obj;
-  const { title, subTitle, titleComponent } = podRingLabel(resourceObj, obj.kind, pods);
+  const { title, subTitle, titleComponent } = hpaControlledScaling
+    ? hpaPodRingLabel(resourceObj, hpa, pods)
+    : podRingLabel(resourceObj, obj.kind, pods);
```

Comment:
```suggestion
    : podRingLabel(resourceObj, kind, pods);
```

Reply:
Ah, curse of review comments lol ... I adjusted content and didn't validate the code around it.